app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,57 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    # @posts = @user.posts
+    @posts = @user.posts.includes(:author)
   end
 
   def show
     @post = Post.find(params[:id])
     @user = User.find(params[:user_id])
-    @comments = @post.comments
+    # @comments = @post.comments
+    # @comments = @post.comments.includes(:post)
```

Comment:
- [OPTIONAL] Same here, please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮